gluten-ut/spark32/src/test/scala/org/apache/spark/sql/catalyst/expressions/GlutenDateExpressionsSuite.scala

```diff
@@ -288,8 +288,8 @@ class GlutenDateExpressionsSuite extends DateExpressionsSuite with GlutenTestsTr
 
   // Modified based on vanilla spark to explicitly set timezone in config.
   test(GlutenTestConstants.GLUTEN_TEST + "DateFormat") {
-    val PST_OPT = Option(PST.getId)
-    val JST_OPT = Option(JST.getId)
+    val PST_OPT = Option("America/Los_Angeles")
```

Comment:
Velox cannot recognize time zone like "-08:00", so use this name instead. Has filed an issue in velox community.